.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Deno",
+  "image": "denoland/deno:latest",
```

Comment:
![Screenshot 2025-01-14 at 17 03 00](https://github.com/user-attachments/assets/cc17d845-a32f-4be2-93bc-318fe39017f9)

Please add sudo and curl into the image so they don't have to be installed on the fly.